crates/router/src/connector/stripe/transformers.rs

```diff
@@ -2007,6 +2022,7 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PaymentIntentRequest {
             setup_future_usage: item.request.setup_future_usage,
             payment_method_types,
             expand: Some(ExpandableObjects::LatestCharge),
+            browser_info,
```

Comment:
can you also add this in setup mandate request?
https://github.com/juspay/hyperswitch/blob/ef1914ec9b75240628ad0c6367499ec063d31e3d/crates/router/src/connector/stripe/transformers.rs#L2047